packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -214,6 +214,20 @@ abstract class WidgetsBindingObserver {
   ///    boilerplate.
   void didChangeTextScaleFactor() { }
 
+  /// Called when the platform's desired brightness changes.
+  ///
+  /// For example, starting in Android Pie, battery saver mode asks all apps to
```

Comment:
I don't understand this and the next paragraph. It seems to be documenting a property, but it's on a method.

Reply:
I replaced these docs as well based on a recommendation from Hans.  Please take another look after the next push.